src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs

```diff
@@ -400,3 +403,17 @@ fn amend_spec(spec: &mut oci::Spec) -> Result<()> {
 
     Ok(())
 }
+
+fn handle_pid_namespace(namespaces: &[oci::LinuxNamespace]) -> bool {
```

Comment:
Could you add a unit test for this new function?

Reply:
fixed

Reply:
I believe the idea of the patch is to copy `handlePidNamespace` from go runtime, in which case additional changes are required:
1. please remove the pidns setting from container spec
2. please keep the code comments of the go runtime as it helps others understand why it is implemented that way

Reply:
@Tim-0731-Hzt Thanks for updating. To be specific, I think the following line of comment is necessary:
```
                // host pidns path does not make sense in kata. Let's just align it with
                // sandbox namespace whenever it is set.
```

Also any explanation why we do not remove the pidns setting from the container spec in runtime-rs?

Reply:
@Tim-0731-Hzt ping, any comment about this question?

> Also any explanation why we do not remove the pidns setting from the container spec in runtime-rs?

Reply:
Ah, I see that it is handled above in line 395. Thanks @Tim-0731-Hzt !